modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -726,6 +726,11 @@ public ObjectDefinition getObjectDefinition(long objectDefinitionId)
 		return objectDefinitionPersistence.findByPrimaryKey(objectDefinitionId);
 	}
 
+	@Override
+	public List<ObjectDefinition> getObjectDefinitions(long objectFolderId) {
+		return objectDefinitionPersistence.findByObjectFolderId(objectFolderId);
```

Comment:
Hey Murilo,
I believe this class already has a method that does this: `getObjectFolderObjectDefinitions()`

Reply:
This method included after I sent this PR. I will use this new method.